src/app/clusters/webrtc-transport-provider-server/WebRTCTransportProviderCluster.h

```diff
@@ -408,6 +408,16 @@ class WebRTCTransportProviderCluster : public DefaultServerCluster
     CheckTurnsOrStunsRequiresUTCTime(const char * commandName,
                                      const Optional<DataModel::DecodableList<ICEServerDecodableStruct>> & iceServers);
 
+    // Templated helper to decode and dispatch commands
+    template <typename DecodableType, typename HandlerFunc>
```

Comment:
Ah I just noticed this isn't `DefaultServerCluster.h` - does your trick to use a member function still work if you put it there?

If you put it in the cluster definition instead of `DefaultServerCluster.h`, it's unlikely people will follow the right pattern - you should put it in `DefaultServerCluster`